### PR TITLE
NewsWeek Project for Microverse

### PR DESCRIPTION
**Pull request by enkog**
### Mock-up page for NewsWeek
_Built with bootstrap_

-We built this page  with:

- CSS

- HTML

- BOOTSTRAP

- FAVICON

- IMAGES

### CODE REVIEW

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Great job guys you have made a tremendous work :clap: :+1: :100:, but you need to make some changes :hammer: :wrench: :toolbox:  to get your project approved :checkered_flag: 

- [x] Most of the articles should have a bold title.

- [x] align the header items in tablet view correctly, they are not well aligned:
image

- [x] Add a little bit of padding on the top of this title:
image

- [x] Remove the padding-bottom from these images, and their title should be bold as I mentioned earlier:
image

- [x] the subscribe section should take two images:

|*Clone*|*Original*|
|:---:|:---:|
| image | image |

- [x] Give a fixed width to your whole page and make sure to center it.

- [x] change your red color to this one #f72210 so it matches the original.

- [x] all of the article images should be wrapped by a link tag.

- [x] on the mobile view you have a broken logo link.

image

- [x] Fix the header on mobile view:
image

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat:

**Comment by dasileker** [comment]
thanks for the review as you see it's all fixed now the padding and we added the <a> tags to images and we forgot to add the reset.css file to the page we add it too  I hope that's all .

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1452 @@
...
+  <link href="https://fonts.googleapis.com/css2?family=Alata:wght@700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" />
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.1/css/bootstrap.min.css"
+    integrity="sha384-VCmXjywReHh4PwowAiWNagnWcLhlEJLA5buUprzK8rxFgeH0kww/aWY76TfkUoSX" crossorigin="anonymous">
+  <link rel="icon" href="favicon.ico">
+
+
+
+  <link rel="stylesheet" href="css/style.css">
+  <!-- <link rel="stylesheet" href="css/reset.css"> -->
```
Comment (andresporras3423) [inline index.html]: Remove commented code

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.footer-logo {
+  object-fit: unset;
+}
+
+#top img {
+  object-fit: unset;
+}
```
Comment (andresporras3423) [inline css/style.css]: Group selectors with the same style definition

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  font-family: Roboto, "Open Sans", "Alata", sans-serif;
+}
+
+.debate article:first-child {
+  border-bottom: 4px solid var(--color-grey);
+}
+
+.debate article:nth-child(2) {
+  border-bottom: 4px solid var(--color-grey);
+}
```
Comment (andresporras3423) [inline css/style.css]: Group selectors with the same style definition

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Nkiruka Oguadinma & Amine Zerradi** 
Good job with the improvements for this project :muscle:  :muscle:
Your page meets most of the requirements :clap: :clap: :clap:  
There are still some issues to solve before you can go ahead:
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition.
- [x] remove commented code in your HTML file
- [x]  Check the footer of your page in small screen size, reduce the size of the Newsweek image:

| Yours | Original |
|:---:|:---:|
|image | image |

- [x] In 728px screen size your "MORE STORIES" sections looks broken:



Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: index.html (hunk 1 above)
Comment (dasileker) [reply index.html]: ok thanks, it's done

File: css/style.css (hunk 2 above)
Comment (dasileker) [reply css/style.css]: ok thanks, it's done

File: css/style.css (hunk 3 above)
Comment (dasileker) [reply css/style.css]: ok thanks, it's done

**Review by SpaYco — APPROVED** [review]
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**